dev_guide/integrating_external_services.adoc

```diff
@@ -55,7 +55,7 @@ requests to the service amongst the service's endpoints.
 Services that represent an external resource do not require associated pods.
 Instead, leave the `Selector` field unset. This represents the external service,
 making the `EndpointsController` ignore the service and allows you to specify
-endpoints manually:
+endpoints manually using IP address:
```

Comment:
You've added "using IP address" above, but the example below doesn't have an IP address anywhere. Should it?